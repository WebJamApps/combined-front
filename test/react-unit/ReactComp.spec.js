import React from 'react';
import { shallow } from 'enzyme';
import HelloWorld from '../../src/components/react-comp';

describe('the App module', () => {
  let RC;
  beforeEach(() => {
    // document.body.innerHTML = '<div><div class="home"></div></div>';
    // element = document.getElementsByClassName('home')[0];
    // console.log(element);
    // console.log(new ReactComponent());
  });
  it('RC text should be "Hello World" ', () => {
    RC = shallow(<HelloWorld />);
    expect(RC.text()).toEqual('Hello World');
  });
  it('one toplevel element', () => {
    expect(RC).toHaveLength(1);
  });

  it('wrapper should match snapshot', () => {
    expect(RC).toMatchSnapshot();
  });
});
