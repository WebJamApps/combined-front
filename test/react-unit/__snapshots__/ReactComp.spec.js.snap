// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the App module wrapper should match snapshot 1`] = `ShallowWrapper {}`;
